ro.go.hmlendea.Duolingo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: ro.go.hmlendea.Duolingo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+separate-locales: false
+command: duolingo-desktop
```

Comment:
```suggestion
command: duolingo-desktop
rename-icon: duolingo
```

it's required to reame the icon including in the desktop file.

Reply:
The desktop file already contains `Icon=duolingo` so it's ok

Reply:
This was ignored and this is why the build is failing.

Reply:
Disclaimer: I didn't mean to be offensive with the word "suggest". I used it because that's what the GitHub interface shows, and is also the preferred form at my workplace (request sounds a bit harsh)